pythonforandroid/graph.py

```diff
@@ -7,49 +7,79 @@
 from pythonforandroid.util import BuildInterruptingException
 
 
-class RecipeOrder(dict):
+def fix_deplist(deps):
+    """ Turn a dependency list into lowercase, and make sure all entries
+        that are just a string become a tuple of strings
+    """
+    deps = [
+        ((dep.lower(),) if not isinstance(dep, (list, tuple))
+         else tuple([dep_entry.lower() for dep_entry in dep]))
```

Comment:
Instead of a single space use this syntax, which is more readable:

```
stuff
if cond
else stuff
for val in iter
```

Reply:
@KeyWeeUsr I changed it but it's a bit finicky, do you like how it looks now?